scripts/baremetal.sh

```diff
@@ -263,12 +263,23 @@ baremetal_ram_stack() {
     kill $SRV_PID
     echo "Done"
 
+    # Extract callgraphs from source files directly
+    if [ -x "$(command -v cflow)" ]; then
```

Comment:
Oh, I didn't know about `command`, I usually use `which` for that, but apparently `which` is not specified by POSIX, while `command` is, so thanks for letting me know about it :)